client/core/core.go

```diff
@@ -4406,8 +4414,11 @@ func (c *Core) connectDEX(acctInfo *db.AccountInfo, temporary ...bool) (*dexConn
 	}
 
 	// Request the market configuration.
-	err = dc.refreshServerConfig() // handleReconnect must too
+	err = dc.refreshServerConfig(serverAPIVers) // handleReconnect must too
 	if err != nil {
+		if errors.Is(err, outdatedClientErr) {
+			sendOutdatedClientNotification(c, dc)
+		}
 		return dc, err
 	}
```

Comment:
Might be less awkward if just do the api check in a separate function, although it makes sense to me that the API version is the first thing checked in the response and a quick fail is best.